Server/src/services/tools/manage_scene.py

```
@@ -27,29 +28,48 @@ async def manage_scene(
                     return default
-                if isinstance(value, int):
-                    return int(value)
-                s = str(value).strip()
-                if s.lower() in ("", "none", "null"):
-                    return default
-                return int(float(s))
+                if s in ("true", "1", "yes", "y", "on"):
+                    return True
+                if s in ("false", "0", "no", "n", "off"):
+                    return False
             except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)